titus-server-master/src/test/java/com/netflix/titus/master/endpoint/v2/rest/ApplicationSlaManagementResourceTest.java

```diff
@@ -147,13 +147,13 @@ public void getAllApplications() {
 
     @Test
     public void getApplicationSlaBySchedulerName() {
-        when(capacityManagementService.getApplicationSLAsForScheduler("kubescheduler")).thenReturn(Arrays.asList(SAMPLE_SLA_MANAGED_BY_KUBESCHEDULER));
```

Comment:
Is there a constant for that?

Reply:
I will create a constant in the next PR.